pom.xml

```diff
@@ -1,866 +1,545 @@
+                  <searchTransitive>true</searchTransitive>
+                  <excludes>
+                    <exclude>org.slf4j:slf4j-log4j12</exclude>
+                    <exclude>org.slf4j:slf4j-log4j13</exclude>
+                    <exclude>org.slf4j:slf4j-jdk14</exclude>
+                    <exclude>org.slf4j:slf4j-jcl</exclude>
+                    <exclude>log4j:log4j</exclude>
+                    <exclude>commons-logging:*</exclude>
+                  </excludes>
+                  <message>Secondary logging frameworks are banned in preference to org.slf4j:*-over-slf4j</message>
+                </bannedDependencies>
+              </rules>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <configuration>
+          <archive>
+            <manifestEntries>
+              <Build-Version>${project.version}</Build-Version>
+              <Build-DateTime>${maven.build.timestamp}</Build-DateTime>
+              <Maven-GroupId>${project.groupId}</Maven-GroupId>
+              <Maven-ArtifactId>${project.artifactId}</Maven-ArtifactId>
+            </manifestEntries>
+          </archive>
+        </configuration>
+      </plugin>
+      <plugin>
+        <artifactId>maven-release-plugin</artifactId>
+        <version>${maven-release-plugin.version}</version>
+        <configuration>
+          <autoVersionSubmodules>true</autoVersionSubmodules>
+          <useReleaseProfile>false</useReleaseProfile>
+          <releaseProfiles>signed</releaseProfiles>
+          <!-- to ensure we only deploy during release:perform -->
+          <goals>deploy</goals>
+          <!-- to ensure that tags are of the right format -->
+          <tagNameFormat>v@{project.version}</tagNameFormat>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-deploy-plugin</artifactId>
+        <version>${maven-deploy-plugin.version}</version>
+        <!-- We disable this in favor of nexus-staging-maven-plugin -->
+        <configuration>
+          <skip>true</skip>
+        </configuration>
+      </plugin>
+      <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>${maven-compiler-plugin.version}</version>
+        <configuration>
+          <release>${java.version}</release>
+          <encoding>${project.build.sourceEncoding}</encoding>
+          <showWarnings>true</showWarnings>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+
+  <profiles>
+    <profile>
+      <id>docker</id>
+      <activation>
+        <file>
+          <exists>Dockerfile</exists>
+        </file>
+      </activation>
+      <properties>
+        <docker.repo/>
+        <docker.image.name>stream-registry</docker.image.name>
+        <docker.image.base>openjdk:11</docker.image.base>
```

Comment:
Would like to use existing, internal Docker image cache. 

Needs documented somewhere how to change the base image, though. E.g.

`mvn clean package -Ddocker.image.base=docker.registry.internal/java:11`